diff --git a/out/src/features/editHelpers.js b/out/src/features/editHelpers.js
index d45e7c8..7f48849 100644
--- a/out/src/features/editHelpers.js
+++ b/out/src/features/editHelpers.js
@@ -116,6 +116,10 @@ function loadEditHelpers(subscriptions) {
     }
     vscode_1.workspace.onDidChangeTextDocument(e => {
         let lastChange = e.contentChanges[0];
+        if (lastChange === undefined) {
+            return;
+        }
+
         let lastChar = lastChange.text;
         let range = lastChange.range;
         let start = range.start;
diff --git a/out/src/features/picatLinter.js b/out/src/features/picatLinter.js
index 2803613..3cbe76c 100644
--- a/out/src/features/picatLinter.js
+++ b/out/src/features/picatLinter.js
@@ -86,4 +86,4 @@ class PicatLinter {
     }
 }
 exports.default = PicatLinter;
-//# sourceMappingURL=picatLinter.js.map
\ No newline at end of file
+//# sourceMappingURL=picatLinter.js.map
